### PR TITLE
Update references and terms

### LINKED ISSUE #4: Pass for normative refs and terms
Need a pass to add terms for use from the OMs.

Need to add in normative references from external to SMPTE documents, such as ISO directives, etc.

### PR DESCRIPTION
**Pull request by SteveLLamb**
First pass update refs, closes #4

### CODE REVIEW

File: doc/main.html [hunk 1]
```diff
@@ -54,10 +65,10 @@ <h2>Style and Form</h2>
     <section id="sec-style-guidelines">
       <h3>Style Guidelines</h3>
       <p>
-        All Engineering Documents should follow the guidelines of the International Organization for Standardization (ISO) Directives with the following exceptions: In the event of a conflict, explicit provisions of <a href="#bib-smpte-standards-operations-manual"></a> and Administrative Guidelines shall prevail. Any variances from the ISO Directives shall be approved by the Director of Engineering or the Standards Vice President.
+        All Engineering Documents should follow the guidelines of the <a href="#bib-iso-directives"></a> with the following exceptions: In the event of a conflict, explicit provisions of <a href="#bib-smpte-standards-operations-manual"></a> and Administrative Guidelines shall prevail. Any variances from the <a href="#bib-iso-directives"></a> shall be approved by the Director of Engineering or the Standards Vice President.
```
Comment (palemieux) [inline doc/main.html:68]: This should be a reference to Part 2 only: SMPTE does not follow Part 1.

File: doc/main.html [hunk 2]
```diff
@@ -149,27 +160,27 @@ <h3>File Formats for Document Submission</h3>
         <dt>XML</dt>
         <dd>
           <p>
-            Any document conforming to W3C Extensible Markup Language (XML) 1.0 (Fifth Edition) or W3C Extensible Markup Language (XML) 1.1 (Second Edition) may be submitted.
+            Any document conforming to <a href="#bib-w3c-xml-10-5ed"></a> or <a href="#bib-w3c-xml-11-2ed"></a> may be submitted.
```
Comment (palemieux) [inline doc/main.html:163]: We should leave undated, i.e. point to the latest edition.

See https://doc.smpte-doc.org/st429-10-private/main/#sec-normative-references

**Review by palemieux — CHANGES_REQUESTED** [review]

**Comment by SteveLLamb** [comment]
@palemieux changes requested via https://github.com/SMPTE/ag-02/pull/14/commits/9f4b3395927ae57c8ea52753d597b7a8b5eeb49b

**Review by palemieux — APPROVED** [review]

**Comment by SteveLLamb** [comment]
Notice to be sent to ST - this PR is under 2-week review, and will be merged on 2023-04-10 if no objections.

**Comment by SteveLLamb** [comment]
Merging as per above, no objections from ST.

**Pull request merged**